### PR TITLE
bug fix: setting sigma_ from covariance matrix eigenvalues 

### PR DESCRIPTION
**Pull request by pbakalov**
the square root was missing in maxent_params.cpp

### CODE REVIEW

File: src/maxent_params.cpp [hunk 1]
```diff
@@ -240,7 +240,7 @@ void ContiParameters::decompose_covariance_matrix(const alps::params& p){
     // below that value)
     int new_ndat_,old_ndat_=ndat();
     for (new_ndat_ =0;new_ndat_<ndat();new_ndat_++)
-      if (var[new_ndat_]>1e-10) break;
+      if (var[new_ndat_]>1e-12) break; //pick cutoff value manually after looking at evalue spectrum
```
Comment (egull) [inline src/maxent_params.cpp]: Please leave the default at 1.e-10 as it was, but insert a parameter that can change this to 1.e-12 (or any other value) as desired by a user.

File: src/maxent_params.cpp [hunk 2]
```diff
@@ -259,7 +259,7 @@ void ContiParameters::decompose_covariance_matrix(const alps::params& p){
       }
     }
     for (int i=0; i<ndat(); ++i) {
-      sigma_[i] = std::abs(var(new_ndat_+i))/static_cast<double>(p["NORM"]);
+      sigma_[i] = std::sqrt(std::abs(var(new_ndat_+i)))/static_cast<double>(p["NORM"]);
```
Comment (egull) [inline src/maxent_params.cpp]: oioioioi... looks consistent with the comment on line 221. @ryanlevy please have a close look at this.

File: src/maxent_simulation.cpp [hunk 3]
```diff
@@ -337,7 +342,7 @@ void MaxEntSimulation::evaluate(){
 
     std::cerr << "spectra"<<sp<< " max backcont diff" <<sp<<  "chi^2 value " <<std::endl;
     std::cerr << "======="<< sp<<" ================="<< sp<< "=========== " <<std::endl;
-    backcontinue(chispec_back_file,specchi,norm,"chispec",chispec_back);
+    backcontinue(chispec_back_file,specchi,norm_fix,"chispec",chispec_back);
```
Comment (egull) [inline src/maxent_simulation.cpp]: Yep, looks like that was a bug, too. Thanks.

**Comment by egull** [comment]
Thanks a lot!! @ryanlevy could you please have a close look at the scaling of the sigma for the covariance matrix and review the changes yourself?

File: src/maxent_params.cpp (hunk 2 above)
Comment (ryanlevy) [reply src/maxent_params.cpp]: Looks like there was a `sqrt` in the ALPS version of the code, must have gotten wiped during the port. Apologies about that, I definitely think this should be there.

**Comment by ryanlevy** [comment]
@pbakalov could you also add a patch that fixes the now failing params test please? Should be as easy as adding a `sqrt` to the last two tests in `test/paramsTest.cpp` (you can check by running `make test`)

**Comment by pbakalov** [comment]
@egull, @ryanlevy have a look. I think it should be fine now.

**Pull request merged**